src/Uno.UI.Tests/Windows_UI_Xaml/FrameworkElementTests/Given_FrameworkElement.cs

```diff
@@ -37,17 +39,30 @@ public void When_LayoutUpdated()
 
 			SUT.Children.Add(item1);
 
-			SUT.Measure(new Windows.Foundation.Size(1, 1));
-			SUT.Arrange(new Windows.Foundation.Rect(0, 0, 1, 1));
+			SUT.Measure(new Size(1, 1));
+			SUT.Arrange(new Rect(0, 0, 1, 1));
 
-			Assert.AreEqual(1, sutLayoutUpdatedCount);
-			Assert.AreEqual(1, item1LayoutUpdatedCount);
+			var sutLayoutUpdate1 = sutLayoutUpdatedCount;
+			var item1LayoutUpdate1 = item1LayoutUpdatedCount;
 
-			SUT.Measure(new Windows.Foundation.Size(2, 2));
-			SUT.Arrange(new Windows.Foundation.Rect(0, 0, 2, 2));
+			SUT.Measure(new Size(2, 2));
+			SUT.Arrange(new Rect(0, 0, 2, 2));
 
-			Assert.AreEqual(2, sutLayoutUpdatedCount);
-			Assert.AreEqual(2, item1LayoutUpdatedCount);
+			var sutLayoutUpdate2 = sutLayoutUpdatedCount;
+			var item1LayoutUpdate2 = item1LayoutUpdatedCount;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'item1LayoutUpdate2'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5037645)